packages/kit/src/core/postbuild/prerender.js

```diff
@@ -459,20 +460,22 @@ async function prerender({ out, manifest_path, metadata, verbose, env }) {
 
 					if (processed_id.includes('[')) continue;
 					const path = `/${get_route_segments(processed_id).join('/')}`;
-					enqueue(null, config.paths.base + path);
+					paths_to_enqueue.push(config.paths.base + path);
 				}
 			}
 		} else {
-			enqueue(null, config.paths.base + entry);
+			paths_to_enqueue.push(config.paths.base + entry);
 		}
 	}
 
 	for (const { id, entries } of route_level_entries) {
 		for (const entry of entries) {
-			enqueue(null, config.paths.base + entry, undefined, id);
+			paths_to_enqueue.push(config.paths.base + entry, undefined, id);
 		}
 	}
 
+	await Promise.all(paths_to_enqueue);
```

Comment:
Why are you calling `Promise.all` with an array of strings?

Reply:
yeah, that was a dumb mistake. fixed